### DISCUSSION TITLE
Zstd compression as alternative to zip

### DISCUSSION THREAD

**Discussion by fooryo** [gildas-lormeau/SingleFile#1488] [Ideas]
I usually manually compress the pages saved with SingleFile using Zstd with maximum compression level and getting very good compression ratios very often, obviously it depends on the content, but that is.

Is it feasible to implement zstd compression via this lib https://github.com/bokuweb/zstd-wasm and give user an option to customize the compression level?

Custom compression level could be great even for zip.

Thanks for you incredible addon

**Comment by gildas-lormeau** [comment]
Thank you for the kind words :)

Basically, I'm not against the idea, but it would require a lot of work to find a way of integrating Zstd into zip.js. For the moment, I don't have the time to tackle this task. 

Exposing an option for setting a ZIP compression level (for self-extracting or pure ZIP formats) should be relatively easy though.
